CMakeLists.txt

```diff
@@ -37,8 +37,11 @@ if( NOT DEFINED CMAKE_MFX_HOME )
   set( CMAKE_MFX_HOME "${CMAKE_HOME_DIRECTORY}/api" )
 endif()
 
-if( "$ENV{MFX_HOME}" STREQUAL "" )
+if(NOT DEFINED $ENV{MFX_HOME} )
   set( ENV{MFX_HOME} ${CMAKE_HOME_DIRECTORY} )
```

Comment:
Do we really need to set `ENV{MFX_HOME}` ourselves? I suggest to drop it. What we should care about is just ${MFX_HOME}.

Reply:
I still await for the response/fix.